cmd/cri-containerd/cri_containerd.go

```diff
@@ -48,6 +48,9 @@ func main() {
 	if err != nil {
 		glog.Exitf("Failed to create CRI containerd service %+v: %v", o, err)
 	}
+	if err := service.Start(); err != nil {
```

Comment:
Found this when running cri validation container test.

I could send another PR to fix this.